lib/nailed/github.rb

```diff
@@ -1,14 +1,75 @@
+#
+# Nailed::Github
+#
 module Nailed
+  
+  #
+  # github helpers
+  #
+  def get_org_repos(github_client, org)
+    all_repos = github_client.org_repos(org)
+    all_repos.map(&:name)
+  end
+
+  def fill_db_after_migration(github_client)
```

Comment:
Well, right now it gets a `github_context` passed, which is confusing. But I see your point, will revert it for now, and split it asap.